src/sentry/api/endpoints/project_index.py

```diff
@@ -46,7 +47,7 @@ def get(self, request: Request) -> Response:
                 queryset = queryset.none()
         elif not (is_active_superuser(request) and request.GET.get("show") == "all"):
             if request.user.is_sentry_app:
-                queryset = SentryAppInstallationToken.objects.get_projects(request.auth)
```

Comment:
It looks like this was the only instance of `SentryAppInstallationToken.objects.get_projects`, so it be safe to delete that method from `sentry_app_installation_token.py`, especially since it looks like we'll want to use your abstraction going forward elsewhere.